### PR TITLE
doc fixes

### PR DESCRIPTION
**Pull request by jeffng-or**
AutoTuner doc fixes

### CODE REVIEW

File: docs/user/InstructionsForAutoTuner.md
Comment (jeffng-or) [inline docs/user/InstructionsForAutoTuner.md:1]: @luarss , should the installation/configuration scripts be run in ./tools/AutoTuner/src/autotuner like it says on line 26 in the doc or in the ORFS directory? Based on the file paths to the scripts, it should be run from ORFS.

Comment (luarss) [reply docs/user/InstructionsForAutoTuner.md:1]: Yes, that is right. We intend to run installation scripts from flow root dir. The message might be more suited for the distributed.py later on in the doc

Comment (jeffng-or) [reply docs/user/InstructionsForAutoTuner.md:1]: OK, thanks. I've updated the note accordingly.

File: docs/user/InstructionsForAutoTuner.md [hunk 1]
```diff
@@ -104,14 +104,15 @@ For Global Routing parameters that are set on `fastroute.tcl` you can use:
 
 ### General Information
 
-The `distributed.py` script uses Ray's job scheduling and management to
+The `distributed.py` script located in `./tools/AutoTuner/src/autotuner` uses [Ray|https://docs.ray.io/en/latest/index.html]'s job scheduling and management to
```
Comment (luarss) [inline docs/user/InstructionsForAutoTuner.md]:
```suggestion
The `distributed.py` script located in `./tools/AutoTuner/src/autotuner` uses [Ray's](https://docs.ray.io/en/latest/index.html) job scheduling and management to
```

Comment (jeffng-or) [reply docs/user/InstructionsForAutoTuner.md]: fixed

**Review by luarss — APPROVED** [review]

**Review by maliberty — APPROVED** [review]

**Pull request merged**